Disable -Werror

Workaround for:

.../tools/Vitis-AI-Runtime/VART/xir/tools/cmd_graph.cpp:112:19: error: loop variable 'x' creates a copy from type 'const std::pair<const std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >' [-Werror=range-loop-construct]
  112 |   for (const auto x : v) {
      |                   ^
.../tools/Vitis-AI-Runtime/VART/xir/tools/cmd_graph.cpp:112:19: note: use reference type to prevent copying
  112 |   for (const auto x : v) {
      |                   ^
      |                   &
cc1plus: all warnings being treated as errors

Signed-off-by: Mark Hatle <mark.hatle@xilinx.com>

diff --git a/tools/Vitis-AI-Runtime/VART/xir/cmake/VitisCommon.cmake b/tools/Vitis-AI-Runtime/VART/xir/cmake/VitisCommon.cmake
index ffde0620d..056b948a5 100644
--- a/cmake/VitisCommon.cmake
+++ b/cmake/VitisCommon.cmake
@@ -16,11 +16,11 @@
 
 set (CMAKE_CXX_STANDARD 14)
 set (CMAKE_C_STANDARD 99)
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -Werror -ggdb -O0 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0 -fno-inline")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -Wall -Werror")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++14 -Wall -Werror")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror")
-set(CMAKE_EXE "${CMAKE_C_FLAGS} -Wall -Werror")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -ggdb -O0 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0 -fno-inline")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -Wall")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++14 -Wall")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+set(CMAKE_EXE "${CMAKE_C_FLAGS} -Wall")
 set(CMAKE_SHARED_LINKER_FLAGS  "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
 
 set(CMAKE_MACOSX_RPATH 1)
